docs/build-with-powerloom/devnet/Glossary.md

```diff
@@ -0,0 +1,81 @@
+
+### Epochs
+
+An Epoch in the context of blockchain typically refers to a range of block heights on the data source blockchain where smart contract applications run, or, simply put, it is a specific period in the blockchain's life.
+
+Each Snapshotter makes observations specific to the state transitions spanning two block heights termed Epochs. Check out our developer docs [here](https://docs.powerloom.io/docs/Protocol/Specifications/Epoch/) to learn more about Epochs.
+
+### RPCs
+
+Remote Procedure Calls (RPCs), allow a program to request a service from another program on a different computer in a network. In the blockchain, an RPC node lets users and applications interact with the blockchain without needing to know network details.
+
+Extracting data from the blockchain state and generating the snapshot can be a complex task. The RPC Helper, defined in utils/rpc.py, has many helper functions to make this process easier. It handles all the retry and caching logic so developers can focus on efficiently building their use cases.
+
+### Prost Chain
+
+The "Prost Chain" refers to the foundational blockchain infrastructure the Powerloom Network utilizes, serving as the anchor chain for storing proofs.
+
+### Protocol State (on Prost Chain)
+
+Protocol State represents the current state of the network, which Snapshotter Peers or anyone else can refer to if the entity needs access to current or historical data references. The protocol state is constantly updated as new epochs are released, and data gets submitted to the protocol state contract residing on the Prost Chain.
+
+### Devnet
+Devnet is a network environment for developers to test and develop blockchain applications. It offers flexibility, speed, and debugging capabilities. Devnet provides a sandbox environment to test ideas like data dashboards.
+
+:::success info
+#### Our Devnet is live!
+**Powerloom’s Devnet is Live!** Access the Devnet and build exciting data applications; you can mint a no-cost NFT slot on the Sepolia Network 🌟. Devnet mint page: [https://devnet-mint.powerloom.dev](https://devnet-mint.powerloom.dev/)
+
+💡 To mint the NFT slot, ETH on the Sepolia network is required. If your balance is insufficient, you can obtain Sepolia ETH through the following link: https://faucet.quicknode.com/drip
+:::
+
+### Data Markets
+
+Data markets are defined by a schema specifying state variables, event log topics, and specific transactions across any smart contract instances that make the decentralized application. eg. A defi protocol like uniswap. In a data market, datasets are data collections, while data sources are the providers of these datasets. 
```

Comment:
> In a data market, datasets are data collections, while data sources are the providers of these datasets.

Data sources are the contracts that are being snapshotted on. The above is definitely not anywhere else in the current docs. [Ref](https://docs.powerloom.io/docs/Protocol/data-sources)